crates/volta-core/src/error/kind.rs

```diff
@@ -429,6 +429,11 @@ pub enum ErrorKind {
         name: String,
     },
 
+    /// Thrown when Volta is unable to find the shimmed command.
+    ShimResolveError {
```

Comment:
As mentioned below, if we make the lookup infallible by handling (i.e. ignoring) the error directly, we shouldn't need a new error type.